example/usps_api/ssl/generate.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
We don't need to include this script as gRPC codebase has details of how to generate them.

Reply:
Done!